test/types/community/cursor.test-d.ts

```diff
@@ -98,3 +98,42 @@ void async function () {
     expectType<number>(item.foo);
   }
 };
+
+interface InternalMeme {
+  _id: string;
+  owner: string;
+  receiver: string;
+  createdAt: Date;
+  expiredAt: Date;
+  description: string;
+  likes: string;
+  private: string;
+  replyTo: string;
+  imageUrl: string;
+}
+
+interface PublicMeme {
+  myId: string;
+  owner: string;
+  likes: number;
+  someRandomProp: boolean; // Projection makes no enforcement on anything
+  // the convenience parameter project<X>() allows you to define a return type,
+  // otherwise projections returns a generic Document
```

Comment:
I don't see a test to show an `expectType<Document[]>` here

Reply:
Added!